.travis.yml

```diff
@@ -155,6 +149,12 @@ jobs:
       rust: stable
       env: TASK=audit
 
+    - name: "lint Rust source using stable toolchain"
```

Comment:
Whoops! I missed this previously.

Could you keep the comments from the former beta task? These are still true, and explain why this _is_ an allowed failure.

Reply:
I can, though I wonder if we may need to edit the comment to better fit the context:

```
    # Allowed if a failure occurs after a new Rust release until the
    # failure is addressed.
```

Reply:
Maybe.

```
# Allowed because a failure may occur after any new Rust version is released.
```

Reply:
Okay; I'm going to try a variant of that:
```
    # Allowed because a failure may occur after a new Rust stable
    # version is released.
```

...along with changing the `allow_failures` annotation to the following:
```
# Allow stable clippy task to fail
```